pkgs/by-name/hx/hx-lsp/package.nix

```
@@ -0,0 +1,25 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage {
+  pname = "hx-lsp";
+  version = "0.2.11";
+
+  src = fetchFromGitHub {
+    owner = "erasin";
+    repo = "hx-lsp";
+    rev = "0.2.11";
+    sha256 = "sha256-wTilbEK3BZehklAd+3SS2tW/vc8WEeMPUsYdDVRC/Ho=";
+  };
+
+  cargoHash = "sha256-dcGInrfWftClvzrxYZvrazm+IWWRfOZmxDJPKwu7GwM=";
+
```

non-blocking: Always a good idea to add this. 
```suggestion
  nativeInstallCheckInputs = [ versionCheckHook ];
```